src/Infrastructure.EntityFramework/AdminConsole/Repositories/OrganizationUserRepository.cs

```diff
@@ -461,9 +461,11 @@ public async Task ReplaceAsync(Core.Entities.OrganizationUser obj, IEnumerable<C
         {
             var dbContext = GetDatabaseContext(scope);
 
-            var existingCollectionUsers = await dbContext.CollectionUsers
-                .Where(cu => cu.OrganizationUserId == obj.Id)
-                .ToListAsync();
+            // Retrieve all collection assignments, excluding DefaultUserCollection
```

Comment:
Non-blocking: Please consider extracting this logic into a method with a descriptive name to replace the comment.

Reply:
I considered it but I think it ended up looking worse because I had to pass in the dataContext as a parameter and the private method would only be used here